cypress/integration/test2.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('ut2019', function() {
+    const languages = ['https://ut2019.voog.com/contact', 'https://ut2019.voog.com/contact-1'];
+    const reply = ['Invalid input', 'Vigane sisend'];
+    it('Submit form unsuccessfully', function() {
+        cy.clearCookies();
+        for (let i = 0; i < languages.length; i++) {
```

Comment:
Which such approach where the for loop is inside the test your test will not run for Estonian language if it already fails in English (the first element in the array). You should loop around the it() to have them as separate tests